protected-api-access-protected-api/README.md

```diff
@@ -0,0 +1,145 @@
+<!-- Keeping yaml frontmatter commented out for now
+---
+# Metadata required by https://docs.microsoft.com/samples/browse/
+# Metadata properties: https://review.docs.microsoft.com/help/contribute/samples/process/onboarding?branch=main#add-metadata-to-readme
+languages:
+- Node.Js
+page_type: sample
+name: "Node.Js application that makes a request to the Graph API from a protected API on behalf of a user"
+description: "This sample Node.Js application shows a confidential client application which calls a protected API which make a request to Microsoft Graph using the on-behalf-of flow."
+products:
+- azure
+- azure-active-directory
+- ms-graph
+urlFragment: ms-identity-docs-code-api-nodejs
+---
+-->
+
+# Node.js | web app | user sign-in, access control (protected routes), protected web API access (Microsoft Graph) | Microsoft identity platform
+
+<!-- Build badges here
+![Build passing.](https://img.shields.io/badge/build-passing-brightgreen.svg) ![Code coverage.](https://img.shields.io/badge/coverage-100%25-brightgreen.svg) ![License.](https://img.shields.io/badge/license-MIT-green.svg)
+-->
+
+This sample Node.Js application demonstrates a confidential client application which calls a protected API which then makes a request to Microsoft Graph via the on-behalf-of flow.
+
+## Prerequisites
+
+- Azure Active Directory (Azure AD) tenant and the permissions or role required for managing app registrations in the tenant.
+- Node.Js 16+
+- An existing Web API
+
+## Setup
+
+### 1. Modify your existing Web API
```

Comment:
Instead of an existing web API, should they not register the web API?

Reply:
Updating Readme, this part shouldn't have been included.